internal/k8s/reconciler/utils_test.go

```diff
@@ -365,27 +365,27 @@ func TestGatewayStatusEqual(t *testing.T) {
 
 func TestGatewayAllowedForSecretRef(t *testing.T) {
 	type testCase struct {
-		name         string
-		fromNS       string
-		toNS         *string
-		toKind       *string
-		toName       string
-		policyFromNS string
-		policyToName *string
-		allowed      bool
+		name        string
+		fromNS      string
+		toNS        *string
+		toKind      *string
+		toName      string
+		grantFromNS string
+		grantToName *string
+		allowed     bool
 	}
 
 	ns1, ns2, ns3 := "namespace1", "namespace2", "namespace3"
 	secret1, secret2, secret3 := "secret1", "secret2", "secret3"
 
 	for _, tc := range []testCase{
-		{name: "unspecified-secret-namespace-allowed", fromNS: ns1, toNS: nil, toName: secret1, policyFromNS: ns1, policyToName: nil, allowed: true},
-		{name: "same-namespace-no-name-allowed", fromNS: ns1, toNS: &ns1, toName: secret1, policyFromNS: ns1, policyToName: nil, allowed: true},
-		{name: "same-namespace-with-name-allowed", fromNS: ns1, toNS: &ns1, toName: secret1, policyFromNS: ns1, policyToName: &secret1, allowed: true},
-		{name: "different-namespace-no-name-allowed", fromNS: ns1, toNS: &ns2, toName: secret2, policyFromNS: ns1, policyToName: nil, allowed: true},
-		{name: "different-namespace-with-name-allowed", fromNS: ns1, toNS: &ns2, toName: secret2, policyFromNS: ns1, policyToName: &secret2, allowed: true},
-		{name: "mismatched-policy-from-namespace-disallowed", fromNS: ns1, toNS: &ns2, toName: secret2, policyFromNS: ns3, policyToName: &secret2, allowed: false},
-		{name: "mismatched-policy-to-name-disallowed", fromNS: ns1, toNS: &ns2, toName: secret2, policyFromNS: ns1, policyToName: &secret3, allowed: false},
+		{name: "unspecified-secret-namespace-allowed", fromNS: ns1, toNS: nil, toName: secret1, grantToName: nil, allowed: true},
+		{name: "same-namespace-no-name-allowed", fromNS: ns1, toNS: &ns1, toName: secret1, grantToName: nil, allowed: true},
+		{name: "same-namespace-with-name-allowed", fromNS: ns1, toNS: &ns1, toName: secret1, grantToName: &secret1, allowed: true},
```

Comment:
@nathancoleman any idea why these two tests would be failing after the rename? I probably just mixed something up but could use another set of 👀

Reply:
It was because I had accidentally deleted `policyFromNS: ns1, ` from both of these tests when renaming `policyToName` to `grantToName`